### ISSUE TITLE
[bug] appearance/theme can't consist with vitepress

### ISSUE THREAD

**Issue by LoTwT** [vuejs/devtools-next#202]
## descriptions

appearance/theme can't consist with vitepress

https://github.com/vuejs/devtools-next/assets/50254496/9f97bf4b-927c-48ed-bd65-2130a8a93a30

## repro

URL: <https://stackblitz.com/edit/vite-yfew9s?file=docs%2F.vitepress%2Fconfig.ts>

## additional context

It works with vitesse after updating deps.

**Comment by brc-dd** [comment]
This needs to be made configurable some way:

https://github.com/vuejs/devtools-next/blob/a6689f3dc2b478ed74ea1f611697fa650a8a7e46/packages/client/src/composables/graph.ts#L55

VitePress uses `storageKey: 'vitepress-theme-appearance'` -- vueuse has `'vueuse-color-scheme'` as default.

**Comment by LoTwT** [comment]
Hi, @brc-dd.
Is there a good way to detect if a site is powered by Vitepress, both in development and production?

**Comment by brc-dd** [comment]
Maybe something like

```ts
/vitepress/i.test(document.querySelector('meta[name=generator]')?.content)
```

**Comment by LoTwT** [comment]
Thank you. ♥️
I think adding `__VITEPRESS__` is a good way. In the future, someone may benefit from it too.

**Comment by brc-dd** [comment]
released `__VITEPRESS__` in vitepress@1.0.0-rc.45

**Comment by alexzhang1030** [comment]
As we discussed in #233, style isolation is intuitive for users. Therefore, we're closing this comment as 'not planned.' I prefer to develop a plugin for this function. Let's revisit this by reopening or creating a new feature request once the devtools have a mature plugin system. Thank you anyway 🫶.
